inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -435,6 +559,12 @@ void ngfunction_2_irv10(pugi::xml_document& doc,
                 NGRAPH_CHECK(i.get_partial_shape().is_static(),
                              "Unsupported dynamic input shape in ", node);
 
+                // WA for LSTMCellv0, peephole input shall not be serialized
+                if (node_type_name == "LSTMCell" && (i.get_index() == 6)) {
```

Comment:
I think that we need to do this thing only for LSTMCell v0, I propose to check the version of the operation too. Or we can use `NodeTypeInfo` to check type and version.

Reply:
Corrected.